### PR TITLE
chore(yarn-cling): rename test packages to real package names

### PR DESCRIPTION
**Pull request by njlynch**
The tests for `yarn-cling` currently reference what were imaginary package names
as placeholders; those package names have now been registered. While this has no
practical impact, switching the package names to be ones we own just so there's
no ambiguity.

An `npm install` is never done on these directories, so the packages chosen really
doesn't matter. Just picked two of our favorites at random. :)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: tools/yarn-cling/test/cling.test.ts [hunk 1]
```diff
@@ -13,20 +13,20 @@ test('generate lock for fixture directory', async () => {
     requires: true,
     version: '1.1.1',
     dependencies: {
-      package2: {
+      'package2': {
```
Comment (nija-at) [inline tools/yarn-cling/test/cling.test.ts:16]: Probably safe to also change this.

Comment (njlynch) [reply tools/yarn-cling/test/cling.test.ts]: Yeah, makes sense. Changed to `cdk`.

**Review by nija-at — APPROVED** [review]
Should this also be applied to `package1`?

**Comment by njlynch** [comment]
> Should this also be applied to package1?

It's not referenced as a dependency in a `package.json` file, so there's no confusion about someone accidentally running `npm install` and getting the wrong thing. That being said, might as well. I renamed it to `jsii`. :)

**Pull request merged**